argus/backend/util/send_email.py

```diff
@@ -0,0 +1,76 @@
+import smtplib
+from typing import List, Set
+
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+from flask import current_app
+from flask import render_template
+
+class Email:
+    #  pylint: disable=too-many-instance-attributes
+    """
+    Responsible for sending emails
+    """
+    _attachments_size_limit = 10485760  # 10Mb = 20 * 1024 * 1024
+    _body_size_limit = 26214400  # 25Mb = 20 * 1024 * 1024
+
+    def __init__(self):
+        self.sender = ""
+        self._password = ""
+        self._user = ""
+        self._server_host = ""
+        self._server_port = ""
+        self._conn = None
+        self._retrieve_credentials()
+        self._connect()
+
+    def _retrieve_credentials(self):
```

Comment:
what is the purpose of this method?

Reply:
Now credentials are written in file it is not securely. In future Secure store could be added. So it will be easy to change the provider of credentials. 
May it could be removed right now

Reply:
yes, private methods can be added/modified without any issues in future. Better not to add upfront.